### ISSUE TITLE
Fix `The following package is missing: hint-http-compression` issue

### ISSUE THREAD

**Issue by alrra** [webhintio/hint#1280]
Steps to reproduce:

* `npm i -g hint@3.2.4`  (node version doesn't matter)
* `hint example.com`
* Respond with `Y` to the following question
    
 ``` 
The following package is missing:
    hint-http-compression
There is a package from your .hintrc file not installed or with an incompatible version. Do you want us to try to install/update them? (Y/n)
```

* You will get:

```
     { Error: Resource http-compression not found
    at ResourceError.ZoneAwareError (/Users/xxxxx/.nvm/versions/node/v10.4.0/lib/node_modules/hint/node_modules/zone.js/dist/zone-node.js:811:33)
    at new AppInsightsAsyncCorrelatedErrorWrapper (/Users/xxxxx/.nvm/versions/node/v10.4.0/lib/node_modules/hint/node_modules/applicationinsights/out/AutoCollection/CorrelationContextManager.js:193:18)
    at new ResourceError (/Users/xxxxx/.nvm/versions/node/v10.4.0/lib/node_modules/hint/dist/src/lib/types/resourceerror.js:6:9)
    at Object.exports.loadResource (/Users/xxxxx/.nvm/versions/node/v10.4.0/lib/node_modules/hint/dist/src/lib/utils/resource-loader.js:245:15)
    at Object.exports.loadHint (/Users/xxxxx/.nvm/versions/node/v10.4.0/lib/node_modules/hint/dist/src/lib/utils/resource-loader.js:284:20)
    at hints.reduce (/Users/xxxxx/.nvm/versions/node/v10.4.0/lib/node_modules/hint/dist/src/lib/config.js:243:41)
    at Array.reduce (<anonymous>)
    at Function.validateHintsConfig (/Users/xxxxx/.nvm/versions/node/v10.4.0/lib/node_modules/hint/dist/src/lib/config.js:242:38)
    at exports.default (/Users/xxxxx/.nvm/versions/node/v10.4.0/lib/node_modules/hint/dist/src/lib/cli/analyze.js:218:55)
    at process._tickCallback (internal/process/next_tick.js:68:7) status: 'NotFound' }
```

----

Possible commits that may introduce the issue (`cd packages/hint && git rev-list hint-v3.2.3...hint-v3.2.4 .`):

80073d8dc5d5f1b1b7fc6b341a22497810684046
ea3ae22f24d06693f0429232f7b35c1d199e763f
293e998e1e3eb1f3716c6a4f941f12ecd8ae9f8f
76c359ce8811545cada9aae175ed9b6539ff4d0e
8bbbf6dfd87a3577b4cc28a0744f6bb0dc175ba7

**Comment by sarvaje** [comment]
@alrra the package `@hint/hint-http-compression` doesn't have a `dist` folder with the code. I think you need to generate a new package for that hint :)

**Comment by alrra** [comment]
I don't know why that happened, maybe some issues with `npm`? 😕

**Comment by alrra** [comment]
Fixed in `@hint/hint-http-compression v1.0.4`.
